webpack.config.js

```diff
@@ -14,29 +15,46 @@ function getLicenseComment(version) {
     ].join('\n').replace(/\$version/g, version);
 }
 
+const plugins = [
+    new ExtractTextPlugin({
+        filename: './release/likely.css',
+        disable: false,
+    }),
+];
+
+if (isProduction) {
+    plugins.concat([
+        new webpack.optimize.DedupePlugin(),
+        new webpack.BannerPlugin(getLicenseComment(packageJson.version)),
+    ]);
+}
+
 module.exports = {
     entry: {
         likely: './source/likely.js',
         // [] is a workaround, see https://github.com/webpack/webpack/issues/300
         'likely-commonjs': ['./source/index.js'],
     },
     output: {
-        filename: '[name].js',
+        filename: './release/[name].js',
         library: 'likely',
         libraryTarget: 'umd',
     },
     module: {
-        loaders: [{
+        rules: [{
```

Comment:
I’m myself working on migration to Webpack 2 in a separate branch. Is it possible to revert the changes related to webpack 2 while keeping everything else working? If yes, could you please do this to prevent conflicts?